src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/transform/action/explain/ExplainTransformRequest.kt

```diff
@@ -0,0 +1,33 @@
+package com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.explain
+
+import org.elasticsearch.action.ActionRequest
+import org.elasticsearch.action.ActionRequestValidationException
+import org.elasticsearch.action.ValidateActions.addValidationError
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import java.io.IOException
+
+class ExplainTransformRequest : ActionRequest {
```

Comment:
instead of having an explicit constructor in line 17 you can just do 

``class ExplainTransform(val transformIds: List<String>) : ActionRequest() {``

once above refactor you can remove line 12 and line 17-19?